purrsoft/src/scenes/Animalute/component.tsx

```diff
@@ -1,11 +1,127 @@
-import { Container } from '@mui/material';
+import React, { useState } from 'react';
+import {
+  Box,
+  Button,
+  TextField,
+  IconButton,
+  Container,
+  Grid,
+  InputAdornment,
+  Typography,
+} from '@mui/material';
+import { Search, Mic } from '@mui/icons-material';
+import { CustomCard } from '../../components/CustomCard';
+import { useTheme } from '@mui/material/styles';
+import { useGetAnimalsQuery } from '../../store';
 
 export const Animalute = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const { data, error, isLoading } = useGetAnimalsQuery();
+
+  const theme = useTheme();
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+  };
+
+  if (isLoading) {
+    return <div>Loading...</div>;
```

Comment:
Here maybe you can add a CircularProgress or LinearProgress Component from MUI instead of just text. 
And maybe make use of the Typography component as well.